packages/block-editor/src/components/block-parent-selector/index.js

```diff
@@ -24,36 +25,34 @@ export default function BlockParentSelector() {
 	const { selectBlock, toggleBlockHighlight } = useDispatch(
 		blockEditorStore
 	);
-	const {
-		parentBlockType,
-		firstParentClientId,
-		shouldHide,
-		hasReducedUI,
-	} = useSelect( ( select ) => {
-		const {
-			getBlockName,
-			getBlockParents,
-			getSelectedBlockClientId,
-			getSettings,
-		} = select( blockEditorStore );
-		const { hasBlockSupport } = select( blocksStore );
-		const selectedBlockClientId = getSelectedBlockClientId();
-		const parents = getBlockParents( selectedBlockClientId );
-		const _firstParentClientId = parents[ parents.length - 1 ];
-		const parentBlockName = getBlockName( _firstParentClientId );
-		const _parentBlockType = getBlockType( parentBlockName );
-		const settings = getSettings();
-		return {
-			parentBlockType: _parentBlockType,
-			firstParentClientId: _firstParentClientId,
-			shouldHide: ! hasBlockSupport(
-				_parentBlockType,
-				'__experimentalParentSelector',
-				true
-			),
-			hasReducedUI: settings.hasReducedUI,
-		};
-	}, [] );
+	const { firstParentClientId, shouldHide, hasReducedUI } = useSelect(
```

Comment:
This diff is the result of just removing the `parentBlockType` prop, which is not used anymore in favor of `useBlockDisplayInformation`.

Reply:
https://github.com/WordPress/gutenberg/pull/30731/files?diff=split&w=1 is always helpful

<img width="228" alt="Screen Shot 2021-04-12 at 12 04 14" src="https://user-images.githubusercontent.com/699132/114377735-47a7c680-9b87-11eb-92c7-df84c4617818.png">